src/overlays/actors/ovl_En_Zl1/z_en_zl1.c

```diff
@@ -27,39 +37,626 @@ const ActorInit En_Zl1_InitVars = {
     (ActorFunc)EnZl1_Update,
     (ActorFunc)EnZl1_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl1/func_80B4AB40.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl1/EnZl1_Init.s")
+ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK0, 0x00, 0x00, 0x39, 0x20, COLSHAPE_CYLINDER },
+    { 0x01, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, 0x00, 0x00, 0x01 },
+    { 20, 46, 0, { 0, 0, 0 } },
+};
+
+UNK_TYPE D_80B4E61C[] = { 0x06007208, 0x06009848, 0x06009C48, 0x06009848 };
+UNK_TYPE D_80B4E62C[] = { 0x06007608 };
+
+typedef struct {
+    u8 unk_00[0x3];
+} u8_3;
+
+typedef struct {
+    u8 unk_00[0xA];
+} u8_A;
+
+typedef struct {
+    AnimationHeader* unk_00[0xA];
+} AH_A;
```

Comment:
These structs shouldn't be necessary as far as I can tell. All the data from this point on (including the `Vec3f`s) looks like it should be inlined in the functions, which will compile to .data that gets copied at the top of the block where they are defined.